examples/workspaces_variables/README.md

```diff
@@ -0,0 +1,55 @@
+## Managing workspaces and variables
+
+This configuration provides an example of how to manage Terraform Cloud / Enterprise workspaces and the variables set on those workspaces using the `tfe` provider.
+
+There are two configurations supplied in this example. The `manager` configuration uses the `tfe` provider to create workspaces and variables. The `managed` configuration is the configuration that will be associated with the workspaces being created. This `managed` workspace represents what would do the actual deployment of the resources of interest, which could represent environments (dev/test/prod), customers, and so on. The usecase is for any configuration that needs to be deployed where the only difference is the variables supplied.
+
+### Initial setup and information gathering
+
+Successful configuration of this example requires several things to be prepared in advance.
+
+* Create or join a Terraform Cloud or Enterprise [organization](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/organizations#creating-organizations).
+* Create a Terraform Cloud or Enterprise token to use with the `tfe` [provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs). While other token types can be used, a [User Token](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/users#api-tokens) is recommended in the beginning so that the provider has the same access as the user experimenting with this configuration.
+* Create a [Github token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) that has access to create and manage OAuth clients. A personal access token should be sufficient. This example uses Github, so the configuration will need to be modified to work with other VCS providers.
```

Comment:
I'm wondering if we should specify the permissions selected based on what's available when establishing a PAT?

Reply:
Last note: let's make sure to specify that the PAT needs full repo scope to avoid an authorization error 👍